tests/conftest.py

```diff
@@ -1514,6 +1514,15 @@ def finalizer():
                         ceph_health_check_base()
                         log.info("Ceph health check passed at teardown")
             except CephHealthException:
+                if config.RUN["skip_reason_test_found"] is None:
+                    squad_name = None
+                    for marker in node.iter_markers():
+                        if "_squad" in marker.name:
+                            squad_name = marker.name
```

Comment:
I think you can break from the loop here once you get squad marker to avoid looping over the other markers.

Reply:
+1